### ISSUE TITLE
Packet 0/75 (SPacketCollectItem)

### ISSUE THREAD

**Issue by Incluuu** [ViaVersion/ViaVersion#3172]
### `/viaversion dump` Output

dump.viaversion.com

### Console Error

```
Put the paste.gg link or text here.
```

### Bug Description

servers version: spigot 1.8.8
![1-b](https://user-images.githubusercontent.com/67196311/207814675-ee5d2f88-5e41-4afc-aa58-a60cff4de728.png)

### Steps to Reproduce

send players between servers on bungeecord

1. /send <player/current/all> <target>

### Expected Behavior

players are kicked

### Checklist

- [X] Via plugins are only running on **EITHER** the backend servers (e.g. Paper) **OR** the proxy (e.g. BungeeCord), **not on both**.
- [X] I have included a ViaVersion dump.
- [X] If applicable, I have included a paste (**not a screenshot**) of the error.
- [X] I have tried the latest build(s) from https://ci.viaversion.com/ and the issue still persists.

**Comment by Barvalg** [comment]
![Error: Invalid or missing dump](https://img.shields.io/badge/Error-Invalid/missing%20dump-red?style=flat-square) </br>

**Comment by HowardZHY** [comment]
please send the full dump link

**Comment by Incluuu** [comment]
> please send the full dump link

I fixed it

**Comment by Elikill58** [comment]
> I fixed link

No, you should do `/viaversion dump` on your server then send the link here

**Comment by HowardZHY** [comment]
https://dump.viaversion.com/9dac5acfc76ee8c782041320c138ea9c45581e3be1f043eead446b827d0dedd0

**Comment by Barvalg** [comment]
![Platform: git--PaperSpigot--%224c7641d%22%20%28MC%3A%201.8.8%29](https://img.shields.io/badge/Platform-git--PaperSpigot--%224c7641d%22%20%28MC%3A%201.8.8%29-3498db?style=flat-square) </br>![ViaVersion (4.5.1): Even with master](https://img.shields.io/badge/ViaVersion%20(4.5.1)-Even%20with%20master-brightgreen?style=flat-square) </br>

**Comment by HowardZHY** [comment]
so only viaver

**Comment by Stevenson031** [comment]
I have the same issue,

**Comment by FormallyMyles** [comment]
Have you tested without ViaVersion, can you provide further info on proxy version / plugins?

**Comment by Incluuu** [comment]
> 

When I tested without viaversion. It worked, didn't kick anyone out. When I updated the server to 1.12.2. Viaversion works and it doesn't kick anyone out. I was using waterfall without any plugins. The problem is with spigot/paper 1.8.8.

**Comment by haroldrc** [comment]
same problem

**Comment by RoccoDev** [comment]
Adding to this, I believe this is caused by many players joining a server (in my case, switching servers on Velocity) at the same time. Players on newer versions might get kicked depending on how many players join the server "after" them.

As reported, this affects 1.19 clients (we originally started noticing this back in September, but that was when we updated ViaVersion) all the way down to at least 1.16. (I don't know whether any other versions are affected, however it's definitely not an issue on 1.8.9, which is to be expected because the server is running on 1.8.8. I also don't know whether earlier versions get different messages.)

**Comment by roccodev** [comment]
I haven't received reports of it this year, so that's a no from me
